Mathlib/Data/Setoid/Partition.lean

```diff
@@ -3,6 +3,9 @@ Copyright (c) 2019 Amelia Livingston. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Amelia Livingston, Bryan Gin-ge Chen, Patrick Massot, Wen Yang, Johan Commelin
 -/
+
+import Mathlib.Algebra.BigOperators.Finprod
+import Mathlib.Data.Set.Card
```

Comment:
Could this result go in Mathlib/Data/Setoid/Partition/Card.lean (moving the current file to Partition/Basic)?